settings/defaults.py

```diff
@@ -4,8 +4,8 @@
 
 DATACITE_USERNAME = ''
 DATACITE_PASSWORD = ''
-DATACITE_URL = 'https://doi.test.datacite.org/'
 DATACITE_PREFIX = '10.70102'  # Datacite's test DOI prefix -- update in production
+DATACITE_URL = 'https://mds.datacite.org/'
```

Comment:
If we're using the production datacite url, do we need to update the prefix as well?

Reply:
Offhand, do you know what the prod prefix is?

Reply:
Hmmm. I do not. Should we keep it the test prefix until we get it into production, and have DevOps add the production url and prefix?

Reply:
It would be easy to find the prod prefix if we think that's the way to go. Just have to check some DOIs on OSF.

Reply:
Got it from matt 👌